Nodejs/Product/Nodejs/NodejsTools.vsct

```diff
@@ -91,6 +91,13 @@
               <CommandFlag>DynamicVisibility</CommandFlag>
           If you do not want an image next to your command, remove the Icon node or set it to <Icon guid="guidOfficeIcon" id="msotcidNoIcon" /> -->
 
+       <Button guid="guidNodeToolsCmdSet" id="cmdidSendFeedback" priority="0x0500" type="Button">
+        <Parent guid="guidSHLMainMenu" id="IDG_VS_HELP_SUPPORT"/>
```

Comment:
Here's how it now looks:

![image](https://cloud.githubusercontent.com/assets/12821956/18403796/c770116c-769b-11e6-8af4-2de328ff360b.png)

I also created a documentation command.